docs/howto/manage-applications.md

```diff
@@ -102,19 +112,21 @@ juju trust <application>
 
 By specifying various flags, you can also use this command to remove trust from an application, or to give an application deployed on a Kubernetes model access to the full Kubernetes cluster, etc.
 
-> See more: {ref}`command-juju-trust`
+```{ibnote}
+See more: {ref}`command-juju-trust`
+```
 
 
 ## Run an application action
 
-> See also: {ref}`action`
->
-> See more: {ref}`manage-actions`
+See {ref}`action`, {ref}`manage-actions`.
```

Comment:
`ibnote` forgotten here?

Reply:
This is the entire content of this section, so here the See bit is not a note but rather a full sentence (note the full stop at the end). Keen eye either way, thanks!!